### PR TITLE
Fix django-stubs-ext missing breaking a production setup

### PR DESCRIPTION
**Pull request by richardebeling**
As django-stubs-ext is required so that `from django_stubs_ext import StrOrPromise` doesn't fail on production, but I also don't want to duplicate the dependency version number across two files.

Opinions? Is there a better way? Maybe by specifying dependencies in pyproject.toml or something?

### CODE REVIEW

**Review by janno42 — APPROVED** [review]
fine for me

**Review by Kakadus — APPROVED** [review]

**Comment by niklasmohrin** [comment]
Couldn't we just do 
```python
from typing import TYPE_CHECKING, Any

if TYPE_CHECKING:
    from django_stubs_ext import StrOrPromise
else:
    StrOrPromise = Any
```

**Review by niklasmohrin — APPROVED** [review]

File: evap/evaluation/tools.py [hunk 1]
```diff
@@ -16,7 +16,11 @@
 from django.utils.datastructures import MultiValueDict
 from django.utils.translation import get_language
 from django.views.generic import FormView
-from django_stubs_ext import StrOrPromise
+
+if TYPE_CHECKING:
+    from django_stubs_ext import StrOrPromise
+else:
+    StrOrPromise = Any
```
Comment (richardebeling) [inline evap/evaluation/tools.py:23]: @niklasmohrin this means that at test runtime, even if `StrOrPromise` would be available with the correct type, we will have it being equal to `Any`. Run-Time Type Checking (typeguard) thus wouldn't catch violations. Maybe it's better to use
```suggestion
try:
    from django_stubs_ext import StrOrPromise
except ImportError:
    assert not TYPE_CHECKING
    StrOrPromise = Any
```
?

Comment (niklasmohrin) [reply evap/evaluation/tools.py:23]: Thats better, yes

**Review by niklasmohrin — APPROVED** [review]

**Pull request merged**